[2.10.1] Backport "Translations update from Weblate"

cfm opened the pull request:
## Status

Ready for review

## Description of Changes

Backport of #7143.

## Testing

* [x] CI is passing
* [x] base is `release/2.10.1`
* [x] Only contains changes from #7143 and #7259
    - *Except* the merge commit 6e3d7bed17537fa8f69f941f21a8eec34e8da113

weblate committed:
Translated using Weblate (Turkish)

Currently translated at 99.4% (352 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Turkish)

Currently translated at 100.0% (353 of 353 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Turkish)

Currently translated at 100.0% (10 of 10 strings)

Co-authored-by: Erdoğan Yağız Şahin <erdy@protonmail.ch>
Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: kwadronaut <kwadronaut@autistici.org>
Co-authored-by: tekrei <tekrei@tutanota.com>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/tr/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/tr/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit e495c1ee47e4c18e3c0be9f941bfe0eb06658e43)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 79c4e12770d926163a0e147e739d9ae46b5581a7)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 501e10fc952577a3fe888eec14d6b19ece641737)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 00f3d8470644ef31930d4848ee6bc9c68a715dbf)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 2b58cb349039550e85c62f7eb0fe300ceb759cbb)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit e52c2c0b0fbb5bb6a3140644037baea9ae182ea4)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Chinese (Traditional))

Currently translated at 96.0% (339 of 353 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Meng Pang Wang <mengpangwang@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/zh_Hant/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 96a9ae5a82611909cf091a0e9244f28c56ee91ea)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 3fc96123c0786b7c6811a15e706d19cac37db274)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 13401a48beedb554cccf5ade2f559e74d9f606e8)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 58af7fb64d67e20d1c08316ed3099271196cc1ad)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 4bdf598545f3b31d7d18997c37f82a68f1a371eb)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit e9792858a31ced855b0b96b42284a41268a07695)

weblate committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (354 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Swedish)

Currently translated at 100.0% (353 of 353 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Jonas Waga <jonas.waga@schibsted.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/sv/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit ede394c2e25105965697c3898364198b4e844017)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 4bf4dc02fa728cbed0f5b11c6c0331e1fcc391e1)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit cb7d2e9764a64766d2bf601f48720e75726229d7)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 2dbd3d210acff76264fb70677a898f7be3295df3)

weblate committed:
Translated using Weblate (Arabic)

Currently translated at 100.0% (354 of 354 strings)

Translated using Weblate (Arabic)

Currently translated at 96.0% (340 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Ahmad Essam <ahmedessamdev@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/ar/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit edc32c39d7f5f30cf07f10486b195b142f9f21bd)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 45cb54aaa6b937c527ed353fcc790c90e805c9f7)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 83a9518a0045621460c0b4500db8df2224d2fc4b)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit c7e45e10452d67b6ba6cdcdadfe1d0a283e6bd53)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit d620c53d7f5bbfed0788a9df455785080de74af9)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 64c42d8f979308c7c0661a3657e2877c14f8269e)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 7533425651abbb52c6ef8697a096bb25b4815eaf)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 165bfec6ac831aaa6e9ccc75cf9bec2cbcc5289b)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Hindi)

Currently translated at 100.0% (10 of 10 strings)

Co-authored-by: Shivam Maurya <shivam.maurya22ug@apu.edu.in>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/hi/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit f95637a5eb05d8e8451c53774a95b4bed2ada4d9)

weblate committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (10 of 10 strings)

Added translation using Weblate (Polish)

Translated using Weblate (Polish)

Currently translated at 100.0% (354 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Adam Rak <kitynska@gmail.com>
Co-authored-by: Cory Francis Myers <cory@freedom.press>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/pl/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/pl/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 32df76c78e4a0459ceee89b53016c92087282a3c)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 4ef4c47f97ad7fa78d195b2a447e4763bb207687)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Icelandic)

Currently translated at 100.0% (353 of 353 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Sveinn í Felli <sv1@fellsnet.is>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/is/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 46fb435aafac0684dd1ead25a062e6a47032c31e)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 1fc360e115f6f3a38944dabe22f8012807ea43ca)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 40d9b5086663f73bd0f4f13e1124546c2f4d2bbe)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit a91f32331a1d98dcabe3d16024b0118d92765de2)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 134b7723d1173ea3509ee413c6a9dabfe22b649c)

weblate committed:
Translated using Weblate (Esperanto)

Currently translated at 100.0% (354 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Pseudolocale add-on <noreply-addon-pseudolocale@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/eo/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 7be7ec5a2dc50245264e07a03cbfdda0b8b65e75)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 5fa0094c1d1f39ea7b3efb2ed6e4dd49eb6f47b5)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit a0824958f9f4133e779e2ae883b4efdf8e6bd884)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 33c28b1fc1bf981ed2be87289855d944d656074e)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit f07843deda57f85f266d10db797781722a55b520)

weblate committed:
Translated using Weblate (Greek)

Currently translated at 95.4% (338 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Dimitris Maroulidis <dmaroulidis@dimitrismaroulidis.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/el/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 9e869229b93e1613f70d5038616102bfe6c28c74)

weblate committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (354 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: Øyvind Bye Skille <oyvind@byeskille.net>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/nb_NO/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 3ed167ba2ac19991435ac1962981daa7ce41ee94)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 8a48d1bcf72725f4f1427ce85602ea85dce33f64)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Russian)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (353 of 353 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Russian)

Currently translated at 100.0% (10 of 10 strings)

Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: julia <juliadeoniziak@wp.pl>
Co-authored-by: neverender <audiolunaire@getgoogleoff.me>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/ru/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/ru/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit ad0a3202f25bf47401261cf0daf53d18dbdfa26d)

weblate committed:
Translated using Weblate (Italian)

Currently translated at 95.1% (337 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Claudio Arseni <claudio.arseni@ubuntu.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/it/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 614bce3311778db180fc7b404e5d7cba1369d3e2)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit daedbdb269842269facc51f85b404979ea7e62e3)

weblate committed:
Translated using Weblate (Finnish)

Currently translated at 100.0% (354 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (353 of 353 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Markus Ottela <oottela@cs.helsinki.fi>
Co-authored-by: Ricky Tigg <ricky.tigg@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: pkm808 <pkm808@protonmail.com>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/fi/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit b5e01bcb16d4e934f5ade4f3484c784f285671a3)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 7dd135067a1edd02c49f5bf9cb14f97b04cee8ca)

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: SecureDrop/SecureDrop
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
(cherry picked from commit 9282bf6e40632b9ab14b8b00b7a15462a4359ea6)

weblate committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Dutch)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Dutch)

Currently translated at 100.0% (354 of 354 strings)

Translated using Weblate (Dutch)

Currently translated at 97.7% (346 of 354 strings)

Translated using Weblate (Dutch)

Currently translated at 96.6% (342 of 354 strings)

Translated using Weblate (Dutch)

Currently translated at 60.0% (6 of 10 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: kwadronaut <kwadronaut@autistici.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/nl/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/nl/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit f0b14df504d0c8daf91d44ba9c4f65bff7a253d2)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit 56e827a65b714c54c4fdb71970fa44df49445f7b)

weblate committed:
Translated using Weblate (French)

Currently translated at 100.0% (354 of 354 strings)

Translated using Weblate (French)

Currently translated at 100.0% (354 of 354 strings)

Translated using Weblate (French)

Currently translated at 100.0% (354 of 354 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: AO Localization Lab <ao@localizationlab.org>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/fr/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
(cherry picked from commit de33cb7b6c4761fb0871c0dec0a2afd3a5dce39f)

cfm committed:
chore: "make update-desktop-files"

(cherry picked from commit 553ac3bafb3470d0b36bf95f0a0e56a9bde4fcb5)

cfm commented:
That was the right *set* of commits.  Let's try the right *sequence* of commits now....

zenmonkeykstop commented on install_files/ansible-base/roles/tails-config/templates/locale/hi/LC_MESSAGES/messages.po:
```diff
@@ -48,4 +37,4 @@ msgstr "SecureDrop स्रोत अंतराफलक"
 #. Icon
 msgid "{{ tails_config_securedrop_dotfiles }}/securedrop_icon.png"
-msgstr ""
+msgstr "सभी क्षेत्रो मे हम एक ही तरह के आइकॉन फाइल प्रयोग करते है"
```
IIRC This should be a path but instead translates to "we use a single icon file in all regions"

zenmonkeykstop replied:
On further examination I'm not sure we're actually using this string - the path is just hardcoded as far as I can tell. If we can confirm that then this is probably not a blocker (but we should remove unused strings from translation so as to avoid confusion and delay).

cfm replied:
Discussed out of band: This winds up being a no-op when the desktop file is [rendered](https://github.com/freedomofpress/securedrop/blob/7e622c654888f6c3418fa1c07509ab4cb044efaf/install_files/ansible-base/roles/tails-config/templates/desktop-journalist-icon.j2#L31) from these translation catalogs, but it's exactly the kind of potentially behavior-altering string change we want to catch.  I'm going to:
1. [x] check Weblate to see why Weblate's own linting didn't catch it;
2. [x] fix this string in Weblate;
3. [x] see if we can mark it read-only to prevent this class of error in the future;
4. [x] bring the fix into `develop`; and then
5. [x] cherry-pick the fix here for final review.

cfm replied:
Weblate is able to enforce the Python-brace formatting we use in general but not the Jinja templating used here.  I've enforced the expected string in <https://github.com/freedomofpress/securedrop/pull/7259#pullrequestreview-2375805844>, which I've cherry-picked in here.  Notice these changes pass linting without `make update-desktop-files`, which confirms that they were a no-op for the rendered desktop file.

zenmonkeykstop reviewed (changes requested):
Changes are confined to the expected subset of commits, but there's a text string in place of a filepath in the Hindi desktop shortcut strings. This should be fixed - users with Hindi setas their language in Tails will at best have a broken icon and at worst undefined behaviours :ghost:

cfm committed:
Translated using Weblate (Hindi)

Currently translated at 100.0% (10 of 10 strings)

Translation: SecureDrop/desktop
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/hi/
(cherry picked from commit 6f34b52c7b7cf20fd3092548d0af8711ee5db791)

cfm committed:
Translated using Weblate (Arabic)

Currently translated at 100.0% (10 of 10 strings)

Translation: SecureDrop/desktop
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/ar/
(cherry picked from commit 7e782370d9cc6b74e872227d59f4bcd90be25829)

zenmonkeykstop commented:
LGTM once CI is good, we should make a note to be sure we include the Hindi and Arabic translator in the acknowledgements, as you're now the most recent updater there :)
